app/models/order_detail.rb

```diff
@@ -260,8 +255,10 @@ def can_be_viewed_by?(user)
       .where.not(statement_id: nil)
   }
 
-  scope :non_reservations, -> { joins(:product).where("products.type <> 'Instrument'") }
-  scope :reservations, -> { joins(:product).where("products.type = 'Instrument'") }
+  # Supports single type or an array of product types
+  scope :for_product_type, ->(product_type) { joins(:product).where(products: { type: product_type }) }
+  scope :plain_orders, -> { for_product_type(["Item", "Service"]) }
```

Comment:
`untimed_orders`, maybe? To contrast with `time_data`? Or `timeless_orders` lol.